.travis.install.cassandra.sh

```diff
@@ -22,7 +22,7 @@ set -xe
 cd "${HOME}"
 
 C_MAJOR="3"
-C_MINOR="7"
+C_MINOR="9"
```

Comment:
Cassandra 3.8 was never released (installation package for 3.8 is the same as 3.9) as it had some serious bugs.